src/components/Navbar/index.css

```diff
@@ -1,40 +1,135 @@
 @import "../../styles/layout.css";
 @import "../../styles/breakpoints.css";
 @import "../../styles/colors.css";
+@import "../../styles/variables.css";
+
+$modal-height: 402px;
 
 .root {
-  display: flex;
-  flex-direction: row;
+  display: grid;
+  align-items: center;
   justify-content: space-between;
+  grid-template-columns: repeat(2, auto);
+  grid-template-rows: 1fr;
 
   padding-top: $spacing-28;
+
+  a {
+    text-decoration: none;
+  }
+
+  @include Breakpoint-mobileOnly {
+    padding-top: $spacing-16;
+  }
 }
 
-.brand {
-  text-decoration: none;
+.root * {
+  z-index: 2;
+}
+
+.root.isOpen {
+  grid-row-gap: $spacing-56;
+  grid-template-rows: repeat(2, auto);
+
+  @include Breakpoint-mobileOnly {
+    max-width: $content-width-mobile;
+
+    padding-right: calc($gutter / 2);
+    padding-left: calc($gutter / 2);
+    margin: 0 auto;
+  }
+}
+
+.brand,
+.brand svg {
```

Comment:
Logo should be center aligned with menus <img width="829" alt="screenshot 2018-12-06 at 09 52 26" src="https://user-images.githubusercontent.com/3351677/49576310-bc75f080-f93c-11e8-923b-26314c800bcf.png">

Reply:
Yes desktop became bugged when I switched to grid, my bad. Fixed